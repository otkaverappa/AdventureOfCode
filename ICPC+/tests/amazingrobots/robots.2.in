5 4
##.#
#..#
#..#
#.X#
####
1
2 2 2 E
5 4
####
#.X#
#..#
#..#
#.##
1
3 2 2 E
