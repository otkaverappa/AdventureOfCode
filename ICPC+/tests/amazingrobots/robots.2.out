10
N
N
N
E
N
W
S
S
S
S
